src/main/java/org/springframework/data/redis/connection/jedis/JedisConverters.java

```diff
@@ -470,16 +468,16 @@ private static byte[] boundaryToBytes(org.springframework.data.domain.Range.Boun
 		byte[] prefix = boundary.isInclusive() ? inclPrefix : exclPrefix;
 		byte[] value = null;
 		Object theValue = boundary.getValue().get();
-		if (theValue instanceof byte[]) {
-			value = (byte[]) theValue;
-		} else if (theValue instanceof Double) {
-			value = toBytes((Double) theValue);
-		} else if (theValue instanceof Long) {
-			value = toBytes((Long) theValue);
-		} else if (theValue instanceof Integer) {
-			value = toBytes((Integer) theValue);
-		} else if (theValue instanceof String) {
-			value = toBytes((String) theValue);
+		if (theValue instanceof byte[] bytes) {
+			value = bytes;
+		} else if (theValue instanceof Double d) {
+			value = toBytes(d);
+		} else if (theValue instanceof Long l) {
+			value = toBytes(l);
+		} else if (theValue instanceof Integer i) {
+			value = toBytes(i);
+		} else if (theValue instanceof String s) {
```

Comment:
This could be `string`. The other variables above could be, for example: `doubleValue`, `longValue`, etc.

Reply:
Okay, I've attached the `value` as suffix to the variables except for string and integer.